docs/features/multidomain.rst

```diff
@@ -0,0 +1,333 @@
+   the same L2 network and try to reconfirm and reuse previous
+   addressing. If multiple neighbouring domains shared the same ESSID,
+   the roaming experience of clients would degrade.
+
+   -  wifi*.ap.ssid
+
+Example config
+--------------
+
+site.conf
+^^^^^^^^^
+
+.. code:: lua
+
+    {
+        site_name = 'Centauri Mesh',
+        site_code = 'centauri',
+        default_domain = 'alpha_centuari',
+        
+        timezone = 'CET-1CEST,M3.5.0,M10.5.0/3',
+        ntp_server = {'ntp1.example.org', 'ntp2.example.org'},
+        regdom = 'DE',
+        
+        wifi24 = {       
+            mesh = {
+                mcast_rate = 12000,
+            },
+        },
+        
+        wifi5 = {
+            mesh = {
+                mcast_rate = 12000,
+            },
+        },
+        
+        mesh_vpn = {
+            mtu = 1312,
+            
+            fastd = {
+                methods = {'salsa2012+umac'},
+            },
+            
+            bandwidth_limit = {
+                enabled = false,
+                egress = 200, -- kbit/s
+                ingress = 3000, -- kbit/s
+            },
+        },
+        
+        autoupdater = {
+            branch = 'stable',
+            
+            branches = {
+                stable = {
+                    name = 'stable',
+                    mirrors = {'http://update.example.org/stable/sysupgrade'},
+                    good_signatures = 2,
+                    pubkeys = {
+                        'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx', -- Alice
+                        'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx', -- Bob
+                        'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx', -- Mary
+                    },
+                },
+            },
+        },
+    }
+
+domains/alpha_centauri.conf
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code:: lua
+
+    {
+        -- multiple codes/names can be defined, the first one is the primary name
+        -- additional aliases can be defined
+        domain_names = {
+            alpha_centauri = 'Alpha Centauri',
+            rigil_kentaurus = 'Rigil Kentaurus',
+            proxima_centauri = 'Proxima Centauri',
+        },
+        
+        -- 32 byte random data in hexadecimal encoding
+        -- Can be generated using: echo $(hexdump -v -n 32 -e '1/1 "%02x"' </dev/urandom)
+        domain_seed = 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
+        
+        -- unique network prefixes per domain
+        prefix4 = '10.xxx.0.0/20',
+        prefix6 = 'fdxx:xxxx:xxxx:xxxx::/64',
+        
+        next_node = {
+            ip4 = '10.xxx.yyy.zzz',
+            ip6 = 'fdxx:xxxx:xxxx:xxxx::xxxx',
+        },
+        
+        wifi24= {
+            ap = {
+                ssid = "alpha-centauri.example.org",
+                channel = 5,
+            },
+            mesh = {
+                id = 'mesh-alpha-centauri',
```

Comment:
we wanted to promote random mesh ids, see https://github.com/freifunk-gluon/gluon/blob/master/docs/site-example/site.conf#L61